src/api/user/src/models/celebrateSchema.js

```diff
@@ -0,0 +1,32 @@
+const { celebrate, Joi, Segments } = require('celebrate');
+
+const get = () =>
+  celebrate({
+    [Segments.PARAMS]: {
+      id: Joi.string().required(),
+    },
+  });
+
+const post = () =>
+  celebrate({
+    [Segments.BODY]: Joi.object().keys({
+      id: Joi.number().required(),
+      firstName: Joi.string().required(),
+      lastName: Joi.string().required(),
+      displayName: Joi.string().default(''),
+      isAdmin: Joi.boolean().default(false),
+      isFlagged: Joi.boolean().default(false),
+      feeds: Joi.array().items(Joi.string()).required(),
+      github: Joi.object({
+        username: Joi.string(),
+        avatarUrl: Joi.string(),
+      }).default({ username: '', avatarUrl: '' }),
+      created: Joi.string(),
+      updated: Joi.string(),
+    }),
+  });
+
+exports.post = post;
+exports.get = get;
+exports.update = post;
```

Comment:
Why do update and delete?  I think you should remove these.

Reply:
Used [here](https://github.com/Seneca-CDOT/telescope/pull/1915/files#diff-dfc53f29574f8dcd8e640e8749550bf7607d40a734ce2814e4961dfd62adf0ccR115) to be explicit, along with `delete`, what do you think of keeping it for the sake of making the middleware easier to understand? I could go both ways.